src/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs

```diff
@@ -204,5 +206,29 @@ private static SafeSecKeyRefHandle ImportKey(ECParameters parameters)
 
             return Interop.AppleCrypto.ImportEphemeralKey(blob, isPrivateKey);
         }
+
+        public unsafe int ImportSubjectPublicKeyInfo(
```

Comment:
I don't understand these public, int-returning methods.  Why are they public (given that they're not override), and why are they returning an int?  I don't see anything using the return value.

Reply:
Uff, should have been `override` I guess. It's weird that I didn't get a warning though. Will investigate.

Reply:
Please mark this method `internal`.  ExportParameters was correctly `internal`, but apparently `ImportParameters` slipped through the cracks.